src/sentry/utils/data_scrubber.py

```diff
@@ -195,3 +195,17 @@ def filter_csp(self, data):
                     querybits.append(chunk)
             query = "&".join("=".join(k) for k in querybits)
             data[key] = urlunsplit((scheme, netloc, path, query, fragment))
+
+
+def ensure_does_not_have_ip(data):
```

Comment:
This was moved out so I don't have to pass a `helper` object around. The method on `helper` I was interested in using didn't even use `self`.